### PR TITLE
733: Add counts to filter links in the translation page.

### LINKED ISSUE #237: Clean up scrutinizer warnings/issues
[Scrutinizer](https://scrutinizer-ci.com/g/GlotPress/GlotPress-WP/) is reporting warnings and issues with the codebase, this is a parent issue to track the cleanup of them.

### LINKED ISSUE #733: Add string counts to the filter links
Hello,

Wanted to suggest adding string counts to the filter links;
screen shot 2017-06-14 at 9 27 53 am

Thanks

### PR DESCRIPTION
**Pull request by toolstack**
Resolves #733.

### CODE REVIEW

File: gp-templates/translations.php [hunk 1]
```diff
@@ -75,18 +75,55 @@
...
+	$untranslated_filters = array(
+		'filters[status]' => 'untranslated',
+		'sort[by]'        => 'priority',
+		'sort[how]'       => 'desc',
+	);
+
+	$filter_links[] = gp_link_get(
+		add_query_arg( $untranslated_filters, $url ),
+		// Translators: %d is the untranslated strings count for the current translation set.
+		sprintf( __( 'Untranslated&nbsp;[%d]&nbsp;(random)', 'glotpress' ), $translation_set->untranslated_count() )
```
Comment (ocean90) [inline gp-templates/translations.php]: We should use `number_format_i18n()` for numbers (and `%s` for the placeholder).

Comment (ocean90) [reply gp-templates/translations.php]: Random was a separate link.

**Review by ocean90 — CHANGES_REQUESTED** [review]
Since we now have the counts, should we hide the links if the count is zero?

**Comment by toolstack** [comment]
I kind of like seeing the zeros, it make it clear there are no fuzzies/waiting/warnings :)

It gives you a sense of accomplishment.

**Review by ocean90 — APPROVED** [review]

**Pull request merged**